### ISSUE TITLE
selector-type-whitelist/blacklist rule

### ISSUE THREAD

**Issue by kristerkari** [stylelint/stylelint#3224]
I don't know if this is possible already, but I would like to whitelist or blacklist certain types of selectors.

e.g.

```js
"selector-type-whitelist": ["class"]
```
valid:
```css
.foo {
  line-height: 1;
}

.foo, .bar {
  line-height: 1;
}
```

invalid: 
```css
#foo {
  line-height: 1;
}

a {
  line-height: 1;
}

input[type=text] {
  line-height: 1;
}
```

**Comment by hudochenkov** [comment]
I think you can achieve this with a combination of [`selector-max-*`](https://stylelint.io/user-guide/rules/#selector-1) rules set to `0`.

**Comment by kristerkari** [comment]
ah yeah, so I can whitelist class selectors by setting all other `selector-max-*` rules than class to `0`.

**Comment by kristerkari** [comment]
Using `selector-max-*` rules allow me to implement what I need. Thanks!
